cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -82,4 +93,36 @@ <h4 class="u-mb5">
     </li>
     {% endfor %}
 </ul>
+
+{% if settings.view_more %}
+  {{ _view_more(settings.view_more) }}
+{% endif %}
+
+{% endmacro %}
+
+{# ==========================================================================
+
+   _view_more()
+
+   ==========================================================================
+
+   Description:
+
+   Creates view more markup when given:
+
+   settings:      An object used to customize the markup.
+
+   settings.href: A string used for the link href.
+
+   settings.text: A string used for the link text.
+
+   ========================================================================== #}
+
+{% macro _view_more(settings) %}
+<a class="jump-link jump-link__underline"
+   href="{{ settings.href or '/' }}">
```

Comment:
Should we really default to the home page? Would it be better to wrap the entire block in an condition based on a url being passed in?